src/plugin.rs

```diff
@@ -233,7 +239,11 @@ async fn select(
             }
           }
           "INT" | "NUMBER" | "INTEGER" | "BIGINT" | "INT8" => {
-            JsonValue::Number(row.get::<i64, usize>(i).into())
+            if let Ok(n) = row.try_get::<i64, usize>(i) {
+              JsonValue::Number(n.into())
+            } else {
+              JsonValue::Null
+            }
           }
           // "JSON" => JsonValue::Object(row.get(i)),
           "BLOB" => JsonValue::Array(
```

Comment:
Should we add the same thing here? Genuine question, don't know if it makes sense for blob.

Reply:
Yeh I think it makes sense because a blob can still be null and would error if it was. Quite busy atm so will try and make the chance tomorrow ☺️